tests/Agent/IntegrationTests/IntegrationTests/Owin/OwinWebApiTests.cs

```diff
@@ -103,7 +103,7 @@ public void Test()
                 () => Assertions.MetricsDoNotExist(unexpectedMetrics, metrics),
                 () => Assert.NotNull(transactionEventWithExternal),
                 () => Assert.Empty(_fixture.AgentLog.GetErrorTraces()),
-                () => Assert.Empty(_fixture.AgentLog.GetErrorEvents())
+                () => Assert.Empty(_fixture.AgentLog.GetErrorEventPayloads())
```

Comment:
Optional, consider using `GetErrorEvents()` instead.   My thinking is that we can widdle down our usage of the GetErrorEventPayloads to the end of removing that method all together.

Reply:
Good idea.  I started out the refactoring process by trying to make minimal changes, but I eventually decided to go in the direction you're suggesting.  Best to be consistent.